server/belga/search_providers.py

```diff
@@ -234,6 +234,10 @@ class Belga360ArchiveSearchProvider(superdesk.SearchProvider):
     def __init__(self, provider):
         super().__init__(provider)
         self.session = requests.Session()
+        self.content_types = {
+            c['label'].lower(): c['_id']
```

Comment:
I think we should use `_id` as a key because Id is unique.

Reply:
see this comment https://github.com/superdesk/superdesk-belga/pull/242#discussion_r454880864